package.json

```diff
@@ -67,6 +68,8 @@
 	},
 	"optionalDependencies": {
 		"@cloudflare/workerd-linux-64": "^1.20240909.0",
+		"@esbuild/darwin-x64": "^0.24.0",
+		"@esbuild/win32-x64": "^0.24.0",
```

Comment:
Why were these needed?

Reply:
I was having errors when running `npm run dev` on both of my computers (with macOS x64 and Windows 11 64). Now, in macOS, I removed the `@esbuild/darwin-x64` package and it is still working 🤔